sap/cli/gcts.py

```diff
@@ -74,6 +74,21 @@ def print_gcts_commit(console, commit_data):
     console.printout('\n   ', commit_data['message'])
 
 
+def get_repository(connection, package):
+    """Get repository corresponding to the PACKAGE"""
+
+    if package.startswith(('http://', 'https://')):
+        repositories = sap.rest.gcts.simple.fetch_repos(connection)
+        repositories = [repo for repo in repositories if repo.url == package]
+
+        if len(repositories) != 1:
```

Comment:
Please, use different message for "not found repository" and for "non unique URL".